src/coverdb.rs

```diff
@@ -72,7 +72,9 @@ impl CoverDB {
                 .values((
                     Cover::Filename.eq(
                         cover.strip_prefix(&self.relative_to).with_context(|| format!("Cover '{}' is not relative to src_dir", cover.display()))?
-                        .to_str().with_context(|| format!("Unable to store filename '{}' in database", cover.display()))?),
+                        .to_str().with_context(|| format!("Unable to store filename '{}' in database", cover.display()))?
+                        .as_bytes()
+                    ),
```

Comment:
How does the OsString look different in the database?
I mean this is not the correct type of the table field. Diesel generated the table macros from the an original table.db.

Reply:
By itself it resulted in the same output, but this allowed me to manipulate the bytes directly to achieve a result identical to usdx.